src/css/archwizard.scss

```diff
@@ -0,0 +1,70 @@
+// color definitions
+$wz-color-default: #E6E6E6 !default;
+$wz-color-current: #808080 !default;
+$wz-color-done: #339933 !default;
+$wz-color-optional: #38ef38 !default;
+$wz-color-editing: #FF0000 !default;
+
+// dot definitions
+$wz-dot-border-width: 2px !default;
+
+// small definitions
+$wz-small-dot-width: 14px !default;
+$wz-small-dot-height: 14px !default;
+
+// big definitions
+$wz-big-dot-width: 50px !default;
+$wz-big-dot-height: 50px !default;
+
+// extra distance between the bottom of the dots and the baseline texts
+$wz-dot-baseline-distance: 0px !default;
+
+// padding between text and baseline, for horizonal navigation bar
+$wz-text-padding-bottom: 10px !default;
+
+// padding between text and baseline, for vertical navigation bar
+$wz-text-margin-left: 15px !default;
+
+// distance between steps, for vertical navigation bar
+$wz-distance-between-steps: 10px !default;
+
+$wz-text-height: 14px !default;
+
+
+aw-wizard {
+  display: flex;
+  justify-content: flex-start;
+
+  &.vertical {

```

Comment:
Is it possible to collect the statements pertaining a navigation bar layout in one block, like it was done for less. Such a css layout has the advantage that it is easier to find all definitions targeting a layout. This in turn reduces possible confusion by accidentally reading the definitions of the other layout.

Reply:
Do you mean something like this (in `archwizard.scss`)?

```
aw-wizard {
  display: flex;
  justify-content: flex-start;

  .wizard-steps {
    top: 0;
    display: flex;
  }
}

aw-wizard.horizontal {
  flex-direction: column;

  .wizard-steps {
    width: 100%;
    flex-direction: row;
  }
}

aw-wizard.vertical {
  flex-direction: row;

  .wizard-steps {
    min-width: calc(100% - 280px);
    width: 80%;
    height: 100%;
    flex-direction: column;
  }
}
```

Reply:
Yes

Reply:
I agree that it looks nicer. Done in 8ee8166.

Reply:
Can we do something similar for `wizard-navigation-bar.scss` too?
I mean the `@if` and `@else if` blocks don't really help readability, right?

Reply:
```
@if ($layout == 'horizontal') {
    @include wz-circle-position-horizontal($width, $height, $border-width);
}
@else if ($layout == 'vertical') {
    @include wz-circle-position-vertical($width, $height, $border-width);
}
```

These if-else blocks use arguments of `wz-define-style` mixin, so the only viable option I see is to group them inside the mixin.

Reply:
Done in 1752ff7. Arguably, it reads much worse.

Reply:
I agree, that in this case it isn't really useful to group the definitions by the layout. I'm fine with reverting 1752ff7 if you think it was better before.

Reply:
Reverted in a97a14c